From c9ac2f47050fabdfab92cfc91de498402193acba Mon Sep 17 00:00:00 2001
From: Maksim Sisov <msisov@igalia.com>
Date: Fri, 23 Mar 2018 10:35:26 +0200
Subject: [PATCH] Fix Assignment had no effect.

ERROR at //build/config/compiler/BUILD.gn:188:12: Assignment had no effect.
|   inputs = []
|            ^
| You set the variable "inputs" here and it was unused before it went
| out of scope.
| See //build/config/BUILDCONFIG.gn:518:3: which caused the file to be included.
|   "//build/config/compiler:afdo",
|   ^-----------------------------

Signed-off-by: Maksim Sisov <msisov@igalia.com>
---
 build/config/compiler/BUILD.gn | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index adf78cbbffe9..0ded07214d71 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -185,7 +185,9 @@ config("compiler") {
   ldflags = []
   defines = []
   configs = []
-  inputs = []
+  if (is_clang) {
+    inputs = []
+  }
 
   # System-specific flags. If your compiler flags apply to one of the
   # categories here, add it to the associated file to keep this shared config
-- 
2.11.0

